cinch/roles/nginx/handlers/main.yml

```diff
@@ -0,0 +1,8 @@
+- name: reload systemd
```

Comment:
yeah, you're actually right. I've changed it in the new PR #116 and it seems to work...